gslib/commands/perfdiag.py

```diff
@@ -117,88 +126,92 @@
   -n          Sets the number of objects to use when downloading and uploading
               files during tests. Defaults to 5.
 
-  -c          Sets the number of processes to use while running throughput
-              experiments. The default value is 1.
+  -c          Sets the number of `processes
+              <https://en.wikipedia.org/wiki/Process_(computing)>`_ to use
+              while running throughput experiments. The default value is 1.
 
-  -k          Sets the number of threads per process to use while running
-              throughput experiments. Each process will receive an equal number
-              of threads. The default value is 1.
+  -k          Sets the number of `threads
+              <https://en.wikipedia.org/wiki/Thread_(computing)>`_ per process
+              to use while running throughput experiments. Each process will
+              receive an equal number of threads. The default value is 1.
 
               NOTE: All specified threads and processes will be created, but may
-              not by saturated with work if too few objects (specified with -n)
-              and too few components (specified with -y) are specified.
+              not by saturated with work if too few objects (specified with ``-n``)
+              and too few components (specified with ``-y``) are specified.
 
-  -p          Sets the type of parallelism to be used (only applicable when
-              threads or processes are specified and threads * processes > 1).
-              The default is to use fan. Must be one of the following:
+  -p          Sets the type of `parallelism
+              <https://en.wikipedia.org/wiki/Parallel_computing>`_ to be used
```

Comment:
I am not sure if the wiki link helps here because when we talk about parallelism here, we are talking about very specific scope, whereas the wiki link is a much broader topic. IMO, that will just distract the user. I think the info below is sufficient to understand what this flag represents.